src/SeisSol.cpp

```diff
@@ -116,16 +117,20 @@ bool seissol::SeisSol::init(int argc, char* argv[]) {
     const auto rlimInKb = rlim.rlim_cur / 1024;
     // Softlimit (rlim_cur) is enforced by the kernel.
     // This limit is pretty arbitrarily set at 2GB.
-    const auto reasonableStackLimit = 2097152; // [kb]
+    constexpr auto reasonableStackLimitInKb = 0x200'000ULL;                    // [kiB]
+    constexpr auto reasonableStackLimit = reasonableStackLimitInKb * 0x400ULL; // [B]
     if (rlim.rlim_cur == RLIM_INFINITY) {
       logInfo(rank) << "The stack size ulimit is unlimited.";
     } else {
-      logInfo(rank) << "The stack size ulimit is " << rlimInKb << "[kb].";
+      logInfo(rank) << "The stack size ulimit is" << rlimInKb
```

Comment:
why not UnitByte.formatPrefix rlimInKb and reasonableStackLimitInKb?

Reply:
It is because the `ulimit` command accepts values in kiB; otherwise I would have preferred that as well